### DISCUSSION TITLE
设置脱敏规则，版本1.9.1

### DISCUSSION THREAD

**Discussion by walter295** [hhyo/Archery#2207] [Q&A] [answered]
根据文档，设置手机号码脱敏规则：
![image](https://github.com/hhyo/Archery/assets/33250926/bbd90959-4f96-4801-8c0c-059eb10ad2e5)
但是实际脱敏效果和预期不一样：
![image](https://github.com/hhyo/Archery/assets/33250926/22e9e030-3993-41f5-b663-34a0dfec014e)
请大家指导一下如何配置脱敏规则

**Comment by nick2wang** [comment, marked as answer]
需要隐藏的组应该填第2个
1 (.{3})
2 (.*)
3 (.{4})

**Reply by walter295** [reply]
我懂了

**Reply by walter295** [reply]
![image](https://github.com/hhyo/Archery/assets/33250926/85ac1b06-64f3-42f0-8770-bfbc7a90dc47)
我想再问问脱敏字段配置的时候，库名、表名是否支持正则匹配，多谢

**Comment by nick2wang** [comment]
> ![image](https://user-images.githubusercontent.com/33250926/253251132-85ac1b06-64f3-42f0-8770-bfbc7a90dc47.png) 我想再问问脱敏字段配置的时候，库名、表名是否支持正则匹配，多谢

这个目前不支持
